### PR TITLE
Disable test_path_is_consistent_regardless_of_base_url

### PR DESCRIPTION
**Pull request by meteorcloudy**
See https://github.com/bazelbuild/bazelisk/pull/465#issuecomment-1805905082

### CODE REVIEW

**Review by katre — APPROVED** [review]

**Comment by fweikert** [comment]
Can you disable the entire test_path_is_consistent_regardless_of_base_url test?

File: bazelisk_test.sh [hunk 1]
```diff
@@ -299,51 +299,52 @@ EOF
...
-  path1="$(grep "PATH is:" log1)"
-  path2="$(grep "PATH is:" log2)"
-
-  [[ -n "${path1}" && -n "${path2}" ]] || \
-      (echo "FAIL: Expected PATH to be non-empty, got path1=${path1}, path2=${path2}"; exit 1)
-
-  [[ "${path1}" == "${path2}" ]] || \
-      (echo "FAIL: Expected PATH to be the same regardless of which mirror was used, got path1=${path1}, path2=${path2}"; exit 1)
-}
+# Disabled due to https://github.com/bazelbuild/bazelisk/pull/465#issuecomment-1805905082
```
Comment (fweikert) [inline bazelisk_test.sh:302]: Might be easier to just comment out line 549 instead :D

Comment (meteorcloudy) [reply bazelisk_test.sh]: Ah ha, didn't realize the test setup!

**Review by katre — APPROVED** [review]

**Review by katre — APPROVED** [review]

**Comment by katre** [comment]
Apparently my approval isn't good enough to auto-merge, probably because I am not a maintainer? :shrug:

**Review by fweikert — APPROVED** [review]

**Pull request merged**